src/footer/footer.yaml

```diff
@@ -0,0 +1,114 @@
+examples:
+- name: default
+  data:
+    {}
+
+- name: with-meta
+  readme: false
+  description: Secondary navigation to meta information relating to the site
+  data:
+    meta:
+      items:
+        - href: '#1'
+          text: 'Item 1'
+        - href: '#2'
+          text: 'Item 2'
+        - href: '#3'
+          text: 'Item 3'
+        - html: Text that explains link <a class="govuk-c-footer__link" href="#">link that was explained</a>
```

Comment:
"Text that explains link" is not read out (tested with Mac Voiceover) when you tab through the links on the page. I wonder if this situation would be better handled by adding a `title` attribute to the link itself? I think adding HTML in this way is okay too but maybe the text here should be at least reworded as adding text in this way doesn't make it accessible.

Reply:
Adding a title attribute won't help, according to https://silktide.com/i-thought-title-text-improved-accessibility-i-was-wrong/

I think this is an abstraction of what's in the footer on GOV.UK:

```
<li>Built by the <a href="https://www.gov.uk/government/organisations/government-digital-service">Government Digital Service</a>
</li>
```

In that specific case I think a link with the text 'Government Digital Service' is clear enough?

Reply:
This is the same behaviour that any list has, since it's not focusable content.